graphql_client_codegen/Cargo.toml

```diff
@@ -17,3 +17,4 @@ quote = "^1.0"
 serde_json = "1.0"
 serde = { version = "^1.0", features = ["derive"] }
 syn = "^1.0"
+thiserror = "2.0.11"
```

Comment:
I'd prefer to avoid adding dependencies that are not 100% needed in this library. I like thiserror for application code, but I wouldn't use it there.

Reply:
Are you possibly confusing 'thiserror' and 'anyhow'? Thiserror targets libraries, not applications. It's a compile-time-only dependency- not exposed in public interface (it generates the boilerplate you would otherwise write by hand)

Reply:
It still adds to the compile times and the dependencies to audit. It's a good crate, but I wouldn't use it for a couple of modules in a library. Arguably, it's not worse than `serde-derive` which we already have in there, but I'm not sure why, we should remove if possible.